Base/Python/slicer/util.py

```diff
@@ -2337,7 +2343,10 @@ def errorDisplay(text, windowTitle=None, parent=None, standardButtons=None, **kw
   """Display an error popup.
 
   If there is no main window then the text is only logged (at error level).
+  Skipped during testing.
   """
+  if slicer.app.testingEnabled():
+    return
```

Comment:
To make sure the test effectively covers code that would otherwise be executed, I suggest to move

```python
  if slicer.app.testingEnabled():
    logging.info("Testing mode is enabled: Skipping error popup [%s]" % windowTitle)
    return
```

just above the call to `messageBox` below.

Doing so would also make sure `logging.error(text)` is called.